DOC-2609: Text input was prevented in form elements in the contents of the editor. (#3585)
tinymce/tinymce-docs: modules/ROOT/pages/7.6.1-release-notes.adoc

[[bug-fixes]]
== Bug fixes

{productname} {release-version} also includes the following bug fixes:

=== Text input was prevented in form elements in the contents of the editor.
// #TINY-11446

> Previously, an issue was identified in Google Chrome where text input was prevented within form elements, resulting in the inability to insert text into these elements.

Reviewer: I wonder if we should include Edge as well since this seems to affect any chromium based browser. Sorry for not including that in the docs notes.

Author: @spocke Yeah that makes sense, I'll add Microsoft Edge as well.

> Previously, an issue was identified in Google Chrome where text input was prevented within form elements, resulting in the inability to insert text into these elements.

Author: ```suggestion
Previously, an issue was identified in Google Chrome and Microsoft Edge where text input was prevented within form elements, resulting in the inability to insert text into these elements.
```

> {productname} {release-version} addresses this issue, by updating the logic for handling text input on non-editable elements to include support for form elements. As a result, text can now be successfully inserted into form elements in Google Chrome.

Author: ```suggestion
{productname} {release-version} addresses this issue, by updating the logic for handling text input on non-editable elements to include support for form elements. As a result, text can now be successfully inserted into form elements in Google Chrome and Microsoft Edge.
```

> Previously, an issue was identified in Google Chrome where text input was prevented within form elements, resulting in the inability to insert text into these elements.
> 
> This has been addressed in {productname} {release-version} by updating the logic for handling text input on non-editable elements to include support for form elements. As a result, text can now be successfully inserted into form elements in Google Chrome.

Reviewer 2: ```suggestion
Previously, an issue was identified in Google Chrome where text input was prevented within form elements, resulting in the inability to insert text into these elements.

{productname} {release-version} addresses this issue, by updating the logic for handling text input on non-editable elements to include support for form elements. As a result, text can now be successfully inserted into form elements in Google Chrome.
```
